### DISCUSSION TITLE
Supported Sensors

### DISCUSSION THREAD

**Discussion by ioukos** [aquaticus/nexus433#31] [Q&A]
This software works like a charme. I use it with 2 sensors for more than 6 monthd with no issue.

As the 2 sensors metionned are not available anymore (Aliexpress, etc.) do you know other supported devices ?

Thank You,

**Comment by aquaticus** [comment]
Hi, I tested ony two sensors mentioned in the text.
I've looked at Aliexpress and indeed Digoo brand seems to be gone. I've found a sensor that looks the same as Digoo -- look for "Baldr". Signle sensor costs 7.6USD (e.g. https://cutt.ly/ekC5r0P) **but I cannot guarntee it uses the same protocol** and works without problems.

**Comment by pmmphd** [comment]
I have just ordered [BALDR Wireless Weather Station Outdoor Sensor](https://www.amazon.com/gp/product/B08FJ413ZN) and [Geevon Wireless Remote Outdoor Sensor](https://www.amazon.com/gp/product/B0894H8FN6). I will let you know if they work.

**Reply by pmmphd** [reply]
I am happy to tell you that the [Geevon Wireless Remote Outdoor Sensor](https://www.amazon.com/gp/product/B0894H8FN6) works just fine. I am still waiting on the other one.

**Reply by pmmphd** [reply]
[BALDR Wireless Weather Station Outdoor Sensor](https://www.amazon.com/gp/product/B08FJ413ZN) seems to be working fine. I like that it uses AA batteries instead of AAA while it has the same form factor as Digoo DG-R8H. Two different things with this one:
- it sends updates every ~160 seconds,
- it blinks twice on each update (first time nexus433 sees the sensor it reports over MQTT the sensor is online twice - one after another), however each update is shown once every 160 seconds.

**Reply by pmmphd** [reply]
I have decided to return the [BALDR Wireless Weather Station Outdoor Sensor](https://www.amazon.com/gp/product/B08FJ413ZN). The range of this sensor seems to be shorter compared to the [Geevon Wireless Remote Outdoor Sensor](https://www.amazon.com/gp/product/B0894H8FN6) and there were a lot of gaps in the collected data.

**Reply by geissbuehler** [reply]
The Geevon sensor you mentioned is currently not available. I have ordered the [EXPLORE SCIENTIFIC 1004H](https://www.amazon.de/gp/aw/d/B08MWRCTWH) that looks the same as the Geevon and I'm glad to let you know that it is also compatible. As receiver I'm using this [RXB6](https://www.makershop.de/module/funk/rxb6-433mhz-antenne/). Both shops are in Germany, but seem to ship worldwide.

**Comment by aquaticus** [comment]
Thanks for update!

**Comment by vadelma-pi** [comment]
I have the [Bresser Temeo Hygro Quadro Thermometer](https://www.amazon.de/gp/product/B0728H9L4Z/) working with the Nexus433 library and RPi3. 
However another one - [Bresser 7020400 Weather Station / Irrigation Sensor](https://www.amazon.de/gp/product/B0093JV1AC) doesn't seem to work with Nexus. 
Interestingly, in my house I catch a weak signal from a 3rd unknown sensor, which seem to show the outside temperature.

**Comment by darianf01** [comment]
Hi
Anyone knows if TFA Dostmann transmitters like TFA 30.3208.02 are using the nexus433 protocol? 
Did somebody manage to receive anything from them?
They do appear to transmit in 433MHz frequency, however I cannot get data (using v2.0.0, tried both in pooling and event mode)
Thanks
-d
LE: corrected the Sensor model from 30.3054 to 30.3208.02

**Comment by markovke** [comment]
[Geevon tx16-2](https://www.amazon.com/Geevon-Outdoor-Wireless-Thermometer-Replacement/dp/B0963DKBQP)  work fine. Thank you for your project
